lib/create-organization-api.js

```diff
@@ -48,11 +49,8 @@ export default function createOrganizationApi ({
    * .catch(console.error)
    */
   function getOrganizationMembership (id) {
-    return http.get('organization_memberships/' + id, {
-      baseURL,
-      headers
-    })
-      .then(response => wrapOrganizationMembership(http, response.data, orgId), errorHandler)
+    return http.get('organization_memberships/' + id, { headers })
+      .then(response => wrapOrganizationMembership(http, response.data, this.sys.id), errorHandler)
```

Comment:
`this.sys.id` isn't correct actually, it's meant to be the organization id.

but since you made the intelligent move to simply pass an org-scoped http client into the organization API, could you please do a similar refactor to the organizationMembership entity and createOrganizationMembershipApi? basically removing the need to pass in an org id, since the http client it receives will already be org scoped (thanks to your smart refactor :) )

Reply:
`this.sys.id` was actually correct, as this is `create-organization-api` the `.getOrganization` method would be invoked with the org object as the `this` context so it'd refer to the org's id.

However, what I didn't realize is that there's no need to even refactor anything but rather just remove orgId from Organization Membership API, the HTTP client we're passing down is already scoped to the org itself as it is the `create-organization-api` after all. So I simplified it and everything seems to be working great!

Reply:
that's exactly what I meant, cool thanks for doing that!